tests/componenttests/server/common/ActionTraits.h

```diff
@@ -38,6 +38,36 @@ struct SetConfiguration
     static constexpr FunctionType m_kFunction{&Stub::setConfiguration};
 };
 
+struct SetStateRequest
```

Comment:
Action names should not contain Request word, so it should be e.g. SetState, SetLogLevels, etc...